apiary-metastore-listener/README.md

```diff
@@ -83,7 +86,9 @@ The following shows an example JSON message representing an "ADD_PARTITION" even
       "eventType": "ADD_PARTITION",
       "dbName": "some_db",
       "tableName": "some_table",
-      "partition": ["col_1", "col_2", "col_3"]
+      "sourceMetastoreUris": "thrift://remote_host:9883",
+      "partitionKeys": ["col_1", "col_2", "col_3"],
```

Comment:
Can we use the same format as [Circus Train](https://github.com/HotelsDotCom/circus-train/tree/master/circus-train-aws-sns)  i.e.

```
"partitionKeys" : {
    "local_date" : "string",
    "local_hour" : "int"
  },
```
So we also have the type information downstream?

Reply:
I almost made this change but I realized that having a JSONObject will not preserve the order which in this case is important.